lib/scarpe/wv/radio.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class WebviewRadio < Scarpe::WebviewWidget
+    attr_reader :text
+
+    def initialize(properties)
+      super(properties)
+      @checked = properties[:checked]
```

Comment:
@checked should be set automatically from properties by super. Similarly, you can just say "super" (with no parens) and it'll automatically pass properties.